vocode/streaming/streaming_conversation.py

```diff
@@ -1004,15 +1004,15 @@ async def send_initial_message(self, initial_message: BaseMessage):
         )
         # TODO: configure if initial message is interruptible
         initial_message_tracker = asyncio.Event()
-        agent_response_event = (
-            self.interruptible_event_factory.create_interruptible_agent_response_event(
-                AgentResponseMessage(message=initial_message),
-                is_interruptible=False,
-                agent_response_tracker=initial_message_tracker,
+        # If it is outbound, update the history with the initial message
+        if self.agent.get_agent_config().call_type == CallType.OUTBOUND:
+            self.agent.update_history(
+                "human", self.transcriptions_worker.buffer.to_message()
             )
-        )
-        self.agent_responses_worker.consume_nonblocking(agent_response_event)
```

Comment:
it looks like `agent.update_history` does create an `AgentResponseMessage`, but with the loss of this line we are no longer consuming it anywhere. What are the consequences of this?

Reply:
`self.agent_responses_worker.consume_nonblocking(agent_response_event)`
The agent responses worker consumes agent response events. So in the update history function we do the equivalent. 
[ref](https://github.com/greymatter-labs/vocode-python-main/blob/840587e57dd410853a4eb9b22f0800b02ec3d0ac/vocode/streaming/agent/state_agent.py#L426C1-L429C14)